independent-projects/tools/codestarts/src/main/java/io/quarkus/devtools/codestarts/CodestartProjectInput.java

```diff
@@ -2,14 +2,13 @@
 
 import static java.util.Objects.requireNonNull;
 
-import io.quarkus.bootstrap.model.AppArtifactKey;
 import io.quarkus.devtools.codestarts.utils.NestedMaps;
 import io.quarkus.devtools.messagewriter.MessageWriter;
 import java.util.Collection;
 import java.util.Map;
 
 public class CodestartProjectInput {
-    private final Collection<AppArtifactKey> dependencies;
+    private final Collection<String> dependencies;
```

Comment:
Why not use `AppArtifactKey` or `AppArtifactCoords`?

Reply:
Because this field can accept both (ga or gav), `AppArtifactCoords` doesn't extend `AppArtifactKey` and `AppArtifactCoords` is not meant to be used without version.